system_metrics_collector/src/system_metrics_collector/utilities.cpp

```diff
@@ -28,10 +29,20 @@ namespace system_metrics_collector
 
 namespace
 {
+
+constexpr const char CPU_LABEL[] = "cpu";
 constexpr const char MEM_TOTAL[] = "MemTotal:";
 constexpr const char MEM_AVAILABLE[] = "MemAvailable:";
 constexpr const char EMPTY_FILE[] = "";
 constexpr const int INVALID_MEMORY_SAMPLE = -1;
+
+double computeCpuTotalTime(const ProcCpuData & measurement1, const ProcCpuData & measurement2)
```

Comment:
`ProcCpuData` should be passed by value since its 2 words long.

Reply:
Done.